src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10DriverPin.Windows.cs

```diff
@@ -42,6 +42,7 @@ public void Dispose()
                 _pin.Dispose();
                 _pin = null;
             }
+			
```

Comment:
Looks like we added some tabs in here by mistake. Please remove when you have a chance. Also, is there a stylecop rule that enforces spaces instead of tabs? If so we should enable that.

Reply:
YesandNo. There's a StyleCop rule that checks that the file uses only one type of intendation and a setting that says which one. But if you have entire files that are wrong, no error will happen. Not sure why this slipped, though. I'll go over everything at the very end again, ok?

Reply:
I'll add checkbox about tabs in the issue and sounds good to me

Reply:
Found that I missed SA1027, which is exactly for this.